packages/cli/commands/init.js

```diff
@@ -82,16 +84,16 @@ exports.command = 'init';
 exports.describe = `initialize ${DEFAULT_HUBSPOT_CONFIG_YAML_FILE_NAME} for a HubSpot account`;
 
 exports.handler = async options => {
-  const { auth: authType = PERSONAL_ACCESS_KEY_AUTH_METHOD.value } = options;
-  const configPath = getConfigPath();
+  const { auth: authType = PERSONAL_ACCESS_KEY_AUTH_METHOD.value, c } = options;
+  const configPath = (c && `${getCwd()}/${c}`) || getConfigPath();
```

Comment:
I think that ``${getCwd()}/${c}`` should be using `path.join()` so that it works as expected in windows.